### ISSUE TITLE
Error: `TODO: set upper limit of mu in linkinv`

### ISSUE THREAD

**Issue by robertour** [zhuwang46/mpath#8]
I am trying to run cv.glmregNB on a metric of duration. I am getting the following error:

```
[1] 1
Error in { : task 34 failed - "TODO: set upper limit of mu in linkinv"
Calls: perform_iterations ... cv.glmregNB -> do.call -> glmregNB -> %dopar% -> <Anonymous>
In addition: Warning messages:
1: In model.matrix.default(Terms, mf, contrasts) :
  non-list contrasts argument ignored
2: In glmregNB(duration ~ gender + age + education + degree +  :
  trace argument is ignored since parallel=TRUE
Execution halted
```

- My guess is that the durations are too skewed (Variance-to-Mean Ratio: 52983.2). I have run this with the log(duration) and that works.

- I see a TODO on the error:  `set upper limit of mu in linkinv`. I am not sure if I can set that myself, and any hint of what this parameter mean would be much appreciated. I did find these parameters in the documentation, but I am not sure what to assign to them:

`start, etastart, mustart, ... arguments for the link{glmreg} function`

- A side issue is that it indicates that the `trace argument is ignored since parallel=TRUE`, but this is not true. I am not running this in parallel. Here is my call:  `cv.glmregNB(formula, data = df_train, nfold = nfold, foldid = foldid, alpha = 1, plot.it=FALSE, parallel = FALSE)`. I am sure it is not running in parallel because there is another issue (posting about it later) that is gone when I set parallel = FALSE. Also, based on the speed, the parameter is working.
